### PR TITLE
LRQA-63529, LPS-134339 | master

### PR DESCRIPTION
**Pull request by JoyceWang08211**
https://issues.liferay.com/browse/LRQA-63529
https://issues.liferay.com/browse/LPS-134339

### CODE REVIEW

**Comment by JoyceWang08211** [comment]
ci:test:security:nocompile

**Comment by JoyceWang08211** [comment]
ci:test:security:nocompile

**Comment by JoyceWang08211** [comment]
ci:test:security:nocompile

**Comment by JoyceWang08211** [comment]
ci:test:bundle

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 1]
```diff
@@ -628,10 +629,11 @@ definition {
 	}
 
 	test EditApplicationScopesAndViewLanguageKeys {
+		property dummy.socket.proxy.disabled = "true";
 		property liferay.dependency.jars = "com.liferay.oauth2.provider.scope.sample.qa.jar";
-		property osgi.module.configuration.file.names = "com.liferay.oauth2.provider.scope.internal.configuration.ConfigurableScopeMapperConfiguration-default.config";
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:632]: Leave the "-default", but replace it with "~default", this is the next format for OSGi configuration files.

**Pull request closed without merging**